assets/src/scripts/components/Cards/Tools.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { Card, ListGroup } from "react-bootstrap";
+import { readValueFromPage } from "../../_utils";
+import listOfTools from "../../data/tools.json";
+
+const codingSystemNameMap: Record<string, string> = {
+  "Pseudo BNF": "bnf",
+  "CTV3 (Read V3)": "ctv3",
+  "Dictionary of Medicines and Devices": "dmd",
+  "ICD-10": "icd10",
+  "SNOMED CT (UK Clinical Edition)": "snomedct",
+};
+
+export default function Tools() {
+  const codingSystemName: string =
+    readValueFromPage("metadata")?.coding_system_name;
+
+  if (!codingSystemName) return null;
+
+  const [tools] = useState(() => {
+    return listOfTools.filter((tool) =>
+      tool.codingTypes.includes(codingSystemNameMap?.[codingSystemName]),
```

Comment:
Don't love `codingTypes` as a name, doesn't seem to fit any of our existing nomenclature.

Reply:
Yeah that should be codingSystem, I'll change that

Reply:
:+1: